extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -94,6 +119,40 @@ class AmpYoutube extends AMP.BaseElement {
   }
 
   /** @private */
+  getPlayerReadyPromise_() {
+    return new Promise(resolve => {
```

Comment:
Though this might be a question of "should we stub" and not "do we need to stub", since this might increase the flakiness of tests failing because of timeouts. What do you think?